### PR TITLE
feat: add font & content to font

### PR DESCRIPTION
**Pull request by MaheshTheDeveloper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This is an extended list including a huge amount of products and services that we find for free!
 
-**Adding a service**
+--Adding a service--
```
Comment (WilliamDavidHarrison) [inline README.md:5]: Please don't update this.

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ This is an extended list including a huge amount of products and services that w
 
 <br>
 
-**Disallowed Services**
+--Disallowed Services--
```
Comment (WilliamDavidHarrison) [inline README.md:15]: Same here.

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]
Please alphabetise everything, as it makes it easier for me. (This includes sections)

Also please don't update sections unless you are adding/removing something from them.

Also on URLs if possible remove the final `/`.

**Comment by WilliamDavidHarrison** [comment]
I will update most of it to match the format myself, as I'm not very busy atm.

Thanks for your help with the project!

**Comment by MaheshTheDeveloper** [comment]
Ok

File: README.md (hunk 1 above)
Comment (MaheshTheDeveloper) [reply README.md]: Done.... I did it to **Adding a service**

File: README.md (hunk 2 above)
Comment (MaheshTheDeveloper) [reply README.md]: Done

**Comment by MaheshTheDeveloper** [comment]
@WilliamDavidHarrison I have fixed all! Will make another comit after fixing removing all the `/`

**Comment by WilliamDavidHarrison** [comment]
> @WilliamDavidHarrison I have fixed all! Will make another comit after fixing removing all the `/`

Ok, thank you!

**Review by WilliamDavidHarrison — DISMISSED** [review]

**Review by WilliamDavidHarrison — APPROVED** [review]

**Pull request merged**